modules/engines/engine-graalium/execution-core/src/main/java/org/eclipse/dirigible/graalium/core/DirigibleJavascriptCodeRunner.java

```diff
@@ -178,8 +182,7 @@ public Module run(Path codeFilePath) {
 
     public Value runMethod(Module codeModule, String methodName, Object... args) {
         return switch (codeModule.moduleType()) {
-            case CJS -> runCjsMethod(codeModule.module(), methodName, args);
-            case ESM -> runEsmMethod(codeModule.module(), methodName, args);
+            case CJS, ESM -> runEsmMethod(codeModule.module(), methodName, args);
```

Comment:
Why is this necessary?

Reply:
This is incorrect. If there is a problem somewhere, it’s definitely not here. CommonJS and ESM are two completely different module systems and there is no standard way for interoperability between them. We shouldn’t treat CommonJS modules as ESMs.

Reply:
Reverted to the original content and execution flow